lib/naughty/layout/legacy.lua

```diff
@@ -400,8 +400,8 @@ function naughty.default_notification_handler(notification, args)
             local action_width = w + 2 * margin
 
             actionmarginbox:buttons(gtable.join(
-                button({ }, 1, function() action:trigger() end),
-                button({ }, 3, function() action:trigger() end)
+                button({ }, 1, function() action:invoke() end),
+                button({ }, 3, function() action:invoke() end)
```

Comment:
Uhm... Since I could never reproduce #2729... is something hiding some error here, or did I just "do it wrong" in trying to reproduce this? In the later case: :+1: and otherwise :-1:

Reply:
This doesn't fix #2729 since I currently cannot reproduce it either. In my case Chromium fallback to its built-in notification for the problematic example even when the register key is set to use the native notification. Firefox doesn't display any notification for that example (not even in `dbus-monitor`, but do for the other examples).

The issue this commit solves was introduced during a rebase. Since the legacy notifications are hard to test (at least without playing with the hierarchy + fake_input), this went unnoticed.